source/features/command-palette-vim-keybindings.tsx

```diff
@@ -0,0 +1,30 @@
+import select from 'select-dom';
+import onetime from 'onetime';
+
+import features from '.';
+
+function commandPaletteKeydown({key, ctrlKey}: KeyboardEvent): void {
+	if (!ctrlKey || (key !== 'n' && key !== 'p')) {
+		return;
+	}
+
+	const commandPalette = select('command-palette');
+
+	if (commandPalette) {
+		const targetKey = key === 'n' ? 'ArrowDown' : 'ArrowUp';
+		commandPalette.dispatchEvent(new KeyboardEvent('keydown', {bubbles: true, key: targetKey, code: targetKey}));
+	}
```

Comment:
Prefer early return:

```suggestion
	if (!commandPalette) {
		return;
	}

	const targetKey = key === 'n' ? 'ArrowDown' : 'ArrowUp';
	commandPalette.dispatchEvent(new KeyboardEvent('keydown', {bubbles: true, key: targetKey, code: targetKey}));
```

Reply:
I may be misunderstanding how `init` works. I saw that you can check for the existence of things in `init`, but does that work for dynamic elements that can appear after a keystroke as is the case with the command palette?

Reply:
So it looks like `command-palette` is always in the dom, 6cb42b9 implements a change so we only listen for `keydown` on the `command-palette` element.